commands/doctor.ts

```diff
@@ -0,0 +1,32 @@
+///<reference path="../../.d.ts"/>
+"use strict";
+
+import os = require("os");
+import future = require("fibers/future");
+
+export class DoctorCommand implements ICommand {
+
+	constructor(private $doctorService: IDoctorService,
+		private $logger: ILogger) { }
+
+	public canExecute(args: string[]): IFuture<boolean> {
+		return future.fromResult(true);
+	}
+
+	public allowedParameters: ICommandParameter[] = [];
+
+	public execute(args: string[]): IFuture<void> {
+		return (() => {
+			var warningsPrinted = this.$doctorService.printWarnings();
+			if (warningsPrinted) {
+				this.$logger.out("Please note that these warnings are just used to help the NativeScript maintainers".bold + os.EOL +
```

Comment:
NativeScript -> `${this.$staticConfig.CLIENT_NAME_ALIAS || this.$staticConfig.CLIENT_NAME}`